pretty.js

```diff
@@ -46,19 +53,40 @@ function _lpadzero (aTarget, aLength, aPadChar) {
 function withSpaces (value, eol) {
   var lines = value.split(/\r?\n/)
   for (var i = 1; i < lines.length; i++) {
-    lines[i] = '    ' + lines[i]
+    lines[i] = indent(4) + lines[i]
   }
   return lines.join(eol)
 }
 
-function filter (value, messageKey, eol) {
+function indent (n) {
+  return Array(n + 1).join(' ')
+}
+
+function filter (value, messageKey, eol, errorLikeObjectKeys) {
+  errorLikeObjectKeys = errorLikeObjectKeys || []
```

Comment:
This ensures clients not passing the 4th argument will not get an error on line #L73.